tests/unistd/dup.c

```diff
@@ -29,7 +57,6 @@ int main() {
   printf("close(f1): %d\n", close(f));
   printf("close(f2): %d\n", close(f2));
   printf("close(f3): %d\n", close(f3));
-  printf("\n");
```

Comment:
In general the tests look that the expected output is contained in the seen output. So it is ok to print more than is expected. I think this might be what's happening here.